src/Agent.Service/Windows/AgentService.cs

```diff
@@ -1,339 +1,53 @@
-﻿using System;
-using System.ComponentModel;
-using System.Diagnostics;
-using System.Globalization;
-using System.IO;
-using System.Reflection;
-using System.Runtime.InteropServices;
-using System.ServiceProcess;
-using System.Threading;
+﻿using System.ServiceProcess;
```

Comment:
There is too much green/red throughout the agent.service project. I'm going to need some additional context so I can understand what is moving versus what is new/changed.

However, I have a more fundamental objection that modifying agent.service is not correct. So it may be better to wait until that objection runs its course.